src/helpers/task.ts

```diff
@@ -0,0 +1,37 @@
+const { CloudTasksClient } = require("@google-cloud/tasks");
+
+const client = new CloudTasksClient();
+
+export async function createHttpTask(payload: string) {
+    const project = process.env.PROJECT_ID;
```

Comment:
Just a thought, if we could name it to projectId for consistency, as you've named it in the previous file.

Reply:
Sure thing, consistency makes sense. Can you point me to the other file where I have used `projectId`?

Reply:
apologies, I see a lot of `project_id`, and got confused with `projectId` 😅

Reply:
hehe no worries at all Ashish, keep asking the good questions 🙇